pipeline/builder_options.go

```diff
@@ -82,7 +93,7 @@ func checkPipelineBuilderOptions(c pipelineBuilderOptions) error {
 	}
 
 	if c.dlq.brokers == nil {
-		log.Warn().Msg("No DLQ Topic is set, all messages be retried forever")
+		log.Warn().Msg("No DLQ Topic is set, all messages will be retried forever.")
```

Comment:
Nit: since the message was changed, you could also change `forever` for `indefinitely`, since "forever" is a very strong promise (PubSub would be 7 days at most, for example)